zetachain/chain.json

```diff
@@ -0,0 +1,95 @@
+{
+  "$schema": "../../chain.schema.json",
+  "chain_name": "zetachain",
+  "status": "live",
+  "network_type": "mainnet",
+  "pretty_name": "ZetaChain",
+  "chain_id": "zetachain_7000-1",
+  "bech32_prefix": "zeta",
```

Comment:
Please register this prefix
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
https://github.com/satoshilabs/slips/pull/1679